gulpfile.js

```diff
@@ -9,13 +9,13 @@ var del = require('del');
 gulp.task('dist', function() {
   gulp.src([ 'src/**/*.coffee' ])
   .pipe(coffee({bare: true}).on('error', gutil.log))
-  .pipe(gulp.dest('./'));
+  .pipe(gulp.dest('./dist'));
 });
 
 gulp.task('coffee', function() {
```

Comment:
Other than an argument difference to coffee() - this task is the same as diff. Could one be eliminated?